js/data-analyst-agent-v2/src/storage.ts

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs';
```

Comment:
nit: `storage` filenames usually refers to some sort of storage implementation that involves both reading from and writing to some source. These functions load static data from the filesystem which I would usually stick in the `utils.ts`. Your call though.